examples/buildless/README.md

```diff
@@ -0,0 +1,25 @@
+# Buildless Example
```

Comment:
Why is it Buildless instead of Snowpack? I don't see any dependencies for "buildless" here.

Reply:
That's the name of the approach basically and I've chosen that way that's because it is basically to show how to deploy an app with "buildless" way. Is that fine to go with the name, `snowpack` and update frameworks @styfle? 🤔